### PR TITLE
[Components with more than 1 service] LPS-184905 Refactor components from dynamic-data-mapping and account-service modules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://issues.liferay.com/browse/LPS-184905

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:forms

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/util/AddDefaultSharedFormLayoutPortalInstanceLifecycleListenerUtil.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import com.liferay.dynamic.data.mapping.form.web.internal.layout.type.constants.DDMFormPortletLayoutTypeConstants;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.model.Layout;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+
+/**
+ * @author Leonardo Barros
+ */
+public class AddDefaultSharedFormLayoutPortalInstanceLifecycleListenerUtil {
+
```
Comment (rafaprax) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/util/AddDefaultSharedFormLayoutPortalInstanceLifecycleListenerUtil.java:26]: Hey @MrJohn1911, rename to `DDMLayoutUtil`, please.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found in the latest relevant run. But there is an SF error. Copyright and author is needed in TestHelper.java

**Comment by MrJohn1911** [comment]
In this last push, I removed the TestHelper class, because it should not be there.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
ci:forward

**Pull request closed without merging**